admin/configuration/default-interface.md

```diff
@@ -0,0 +1,27 @@
+# Default user interface
+
+The 2i2c hubs offer the following user interfaces by default:
+
+## Jupyter Notebook (Classic)
+
+What many people mean when they say 'Jupyter', this familiar interface is used
+by default. Document oriented, no-frills, and well known by a lot of people.
+
+## JupyterLab
+
+
+```{figure} ../../images/jupyterlab.png
+:alt: JupyterLab layout
+```
+
+[JupyterLab](https://github.com/jupyterlab/jupyterlab) is a more modern version of the classic Jupyter notebook from
+the Jupyter project. It is more customizable and better supports advanced use cases - particularly around [dask](https://dask.org). Many
+research organizations use this.
+
+## RStudio
+
+```{figure} ../../images/rstudio.png
+:alt: RStudio
+```
+
+[RStudio](https://rstudio.com) is a popular web-based datascience IDE focused on the R language. 
```

Comment:
Similarly here, I think we should avoid describing things in terms of their popularity. I'd just say "An IDE for R, created by the RStudio company." or something like that. That's [how they describe it](https://www.rstudio.com/products/rstudio/)